### PR TITLE
WINC-1113: Introduce feature annotations to CSV

### PR DESCRIPTION
**Pull request by alinaryan**
Adds infrastructure-related annotations: 'features.operators.openshift.io/*' to the operator CSV file, and sets each to string 'true' or 'false' for the WMCO bundle to pass the new CVP Operator Infrastructure Feature Test in the Operator Bundle Image Validation pipeline.

### CODE REVIEW

File: bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -8,6 +8,16 @@ metadata:
     certified: "false"
     createdAt: REPLACE_DATE
     description: An operator that enables Windows container workloads on OCP
+    features.operators.openshift.io/disconnected: "false"
+    features.operators.openshift.io/fips-compliant: "false"
+    features.operators.openshift.io/proxy-aware: "true"
+    features.operators.openshift.io/cnf: "false"
+    features.operators.openshift.io/cni: "false"
```
Comment (saifshaikh48) [inline bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml:15]: don't we build, ship, and configure cni plugins?

Comment (alinaryan) [reply bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml]: yes, thanks for catching. fixed

**Pull request merged**